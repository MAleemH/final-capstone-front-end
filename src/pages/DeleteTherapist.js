import React from 'react';
import { Link } from 'react-router-dom';
import Navigation from '../components/Navigation';
import '../css/DeleteTherapist.css';
import therapistImg from '../img/therapist.jpg';
import trashImg from '../img/trash.png';

function DeleteTherapist() {
  return (
    <main className="delete_page_main">
      <section className="delete_page_nav">
        <Navigation />
      </section>

      <section className="delete_page_content">
        <header>
          <h1>Starred Therapists</h1>

          <p className="hide1">Please select a therapist</p>

          <div className="search_box">
            <fieldset className="fieldset_border_none">
              <input className="input_name" aria-label="Input Name" type="text" placeholder="Search by name" />
            </fieldset>
          </div>
        </header>

        <div className="delete_grids">

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

          <div className="delete_grids_box">
            <Link className="td_none" to="/therapistdetails">
              <article>
                <figure className="therapy_figure">
                  <img src={therapistImg} alt="" />
                </figure>

                <h4>MARCUS DAVID</h4>

                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit, id.</p>

                <div className="delete_page_socials">
                  <figure className="delete_page_social_icons">
                    <button type="button">
                      <img src={trashImg} alt="" />
                    </button>
                  </figure>
                </div>

              </article>
            </Link>
          </div>

        </div>

      </section>

    </main>
  );
}

export default DeleteTherapist;
