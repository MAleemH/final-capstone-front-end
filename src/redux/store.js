import { configureStore } from '@reduxjs/toolkit';
import therapyReducer from './Therapy/therapySlice';
import userReducer from './User/userSlice';

const store = configureStore({
  reducer: {
    therapy: therapyReducer,
    user: userReducer,
  },
});

export default store;
