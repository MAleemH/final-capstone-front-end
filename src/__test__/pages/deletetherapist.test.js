import React from 'react';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import store from '../../redux/store';
import DeleteTherapist from '../../pages/DeleteTherapist';

it('Delete Therapist page renders correctly', () => {
  const myTherapists = store.getState().therapy.therapists;
  const tree = renderer.create(
    <Provider store={store}>
      <MemoryRouter>
        <DeleteTherapist myTherapists={myTherapists} />
      </MemoryRouter>
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
