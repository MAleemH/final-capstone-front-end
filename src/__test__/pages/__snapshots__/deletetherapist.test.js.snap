// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Therapist page renders correctly 1`] = `
<main
  className="delete_page_main"
>
  <section
    className="delete_page_nav"
  >
    <main>
      <section
        className="home_nav_section"
      >
        <header
          className="home_logo"
        >
          <figure>
            <a
              className="td_none"
              href="/homepage"
              onClick={[Function]}
            >
              <img
                alt=""
                src="logo.png"
              />
            </a>
          </figure>
        </header>
        <nav
          className="list_style_none home_nav"
        >
          <li>
             
            <a
              activeClassName="active"
              className="td_none"
              href="/homepage"
              onClick={[Function]}
            >
              HOME
            </a>
          </li>
          <li>
             
            <a
              activeClassName="active"
              className="td_none"
              href="/appointments"
              onClick={[Function]}
            >
              APPOINTMENTS
            </a>
          </li>
          <li>
             
            <a
              activeClassName="active"
              className="td_none"
              href="/book"
              onClick={[Function]}
            >
              NEW APPOINTMENT
            </a>
          </li>
          <li>
            <button
              className="logout_btn"
              onClick={[Function]}
              type="button"
            >
              LOGOUT
            </button>
          </li>
        </nav>
        <footer
          className="home_nav_footer"
        >
          <figure
            className="nav_social_icons"
          >
            <a
              className="td_none"
              href="/"
              onClick={[Function]}
            >
              <img
                alt=""
                src="facebook.png"
              />
            </a>
          </figure>
          <figure
            className="nav_social_icons"
          >
            <a
              className="td_none"
              href="/"
              onClick={[Function]}
            >
              <img
                alt=""
                src="twitter.png"
              />
            </a>
          </figure>
          <figure
            className="nav_social_icons"
          >
            <a
              className="td_none"
              href="/"
              onClick={[Function]}
            >
              <img
                alt=""
                src="linkedin.png"
              />
            </a>
          </figure>
          <figure
            className="nav_social_icons"
          >
            <a
              className="td_none"
              href="/"
              onClick={[Function]}
            >
              <img
                alt=""
                src="instagram.png"
              />
            </a>
          </figure>
          <figure
            className="nav_social_icons"
          >
            <a
              className="td_none"
              href="/"
              onClick={[Function]}
            >
              <img
                alt=""
                src="snapchat.png"
              />
            </a>
          </figure>
        </footer>
      </section>
      <section
        className="home_nav_section_mobile"
      >
        <header
          className="home_logo margin_left1"
        >
          <button
            aria-label="button"
            onClick={[Function]}
            type="button"
          >
            <img
              alt=""
              src="menu.png"
            />
          </button>
        </header>
      </section>
    </main>
  </section>
  <section
    className="delete_page_content"
  >
    <header>
      <h1>
        Starred Therapists
      </h1>
      <p
        className="hide1"
      >
        Please select a therapist
      </p>
      <div
        className="search_box"
      >
        <fieldset
          className="fieldset_border_none"
        >
          <input
            aria-label="Input Name"
            className="input_name"
            onChange={[Function]}
            placeholder="Search by name"
            required={true}
            type="text"
            value=""
          />
        </fieldset>
      </div>
    </header>
    <div>
      <div
        className="home_grids"
      />
    </div>
  </section>
</main>
`;
